components/Main/Workouts/Workouts.js

```diff
@@ -1,29 +1,67 @@
 import { StyledWorkouts, StyledWorkoutsHeader } from "./StyledWorkouts";
 import Image from "next/image";
-import Link from "next/link";
+import InputWorkout from "./InputWorkout/InputWorkout";
+import { useState } from "react";
+import React from "react";
+
+const Workouts = ({ workouts, onAddWorkout, onDelete }) => {
+  const [showInput, setShowInput] = useState(false);
+
+  function toggleShow() {
+    setShowInput(true);
+  }
+
+  async function handleSubmit(event) {
+    event.preventDefault();
+    event.stopPropagation();
+    const name = event.target.elements.name.value;
+    const newWorkout = {
+      name: name,
+    };
+
+    await fetch("/api/workouts", {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify(newWorkout),
+    });
+    onAddWorkout(newWorkout);
```

Comment:
hier wieder das gleiche wie letztens, ich würde den POST hier rausnehmen und das die `handleAddWorkout` in der _app.js übernehmen lassen :)